packages/flutter/lib/src/widgets/page_storage.dart

```diff
@@ -130,6 +130,24 @@ class PageStorageBucket {
 }
 
 /// A widget that establishes a page storage bucket for this widget subtree.
+///
+/// {@tool sample}
```

Comment:
```suggestion
/// {@tool snippet}
```

It looks like the analyzer is failing because `sample`s are expected to work embedded in the full application template. Using `snippet` instead should work.